### ISSUE TITLE
Accessibility: paper-icon-button not focusable with tab key

### ISSUE THREAD

**Issue by erictakahashi** [PolymerElements/paper-tabs#228]
### Description
I have 2 questions about accessibility, I'd like to know if there is a reason for:
- `paper-icon-button` not focusable using keyboard **tab** key
- `paper-icon-button` doesn't have an **alt** text
